### DISCUSSION TITLE
Duplicate drawing on second monitor.

### DISCUSSION THREAD

**Discussion by jn213x** [geovens/gInk#122] [Ideas]
Hi,  It would be great if this application could draw a duplicate image on an extended display.  So what you draw on monitor 1 is duplicated on monitor 2.  This would be used in a streaming setting so you can have monitor 1 as an outgoing feed and monitor 2 as a green or chroma screen output to a mixing device.

**Comment by pubpub-zz** [comment]
Hi @jn213x, You may be interesting in this discussion:
https://github.com/pubpub-zz/ppInk/discussions/128
